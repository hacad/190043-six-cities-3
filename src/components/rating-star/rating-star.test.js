import React from "react";
import renderer from "react-test-renderer";

import RatingStar from "./rating-star.jsx";

it(`Rating Star correctly renders after relaunch`, () => {
  const ratingStar = renderer.create(
      <RatingStar
        title="perfect!"
        value={5}
      />
  ).toJSON();

  expect(ratingStar).toMatchSnapshot();
});
