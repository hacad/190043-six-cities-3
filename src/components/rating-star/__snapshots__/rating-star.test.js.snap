// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rating Star correctly renders after relaunch 1`] = `
Array [
  <input
    className="form__rating-input visually-hidden"
    id="5-stars"
    name="rating"
    required={true}
    type="radio"
    value={5}
  />,
  <label
    className="reviews__rating-label form__rating-label"
    htmlFor="5-stars"
    title="perfect!"
  >
    <svg
      className="form__star-image"
      height="33"
      width="37"
    >
      <use
        xlinkHref="#icon-star"
      />
    </svg>
  </label>,
]
`;
